test/e2e/features/basic.feature

```diff
@@ -22,165 +22,69 @@ Feature: Basic test
         And stdout should contain "delete"
         And stdout should contain "status"
         And stdout should contain "Flags:"
-        And stdout should contain
-            """
-            Use "crc [command] --help" for more information about a command.
-            """
 
     @darwin @linux @windows
     Scenario: CRC status
         When executing crc status command fails
-        Then stderr should contain
-            """
-            Machine does not exist. Use 'crc start' to create it
-            """
+        Then stderr should contain "Machine does not exist. Use 'crc start' to create it"
 
     @linux
-    Scenario: CRC setup on Linux
+    Scenario: CRC setup
         When executing "crc setup --check-only" fails
         Then executing crc start command fails
         And executing single crc setup command succeeds
 
-    @linux
```

Comment:
Why we are removing this test?

Reply:
I removed it because it cannot run in parallel with the rest of the scenarios in the Feature. Tagging would make it hard to read & cluttered if I included it in the next Scenario. So it's going to go in the integration test. I'll list it in the issue of things to move to integration.